spec/core/ics-026-routing-module/README.md

```diff
@@ -41,6 +41,20 @@ The functions `newCapability` & `authenticateCapability` are defined as in [ICS
 
 Modules must expose the following function signatures to the routing module, which are called upon the receipt of various datagrams:
 
+#### **ChanOpenInit**
+
+ChanOpenInit will verify that the relayer-chosen parameters
+are valid and perform any custom INIT logic.
+It may return an error if the chosen parameters are invalid
+in which case the handshake is aborted.
+If the provided version string is non-empty, ChanOpenInit should return
+the version string if valid or an error if the provided version is invalid.
+If the version string is empty, ChanOpenInit is expected to
+return a default version string representing the version(s)
+it supports.
+If there is no default version string for the application,
+it should return an error if provided version is empty string.
+
```

Comment:
```suggestion
onChanOpenInit will verify that the relayer-chosen parameters
are valid and perform any custom INIT logic.
It may return an error if the chosen parameters are invalid
in which case the handshake is aborted.
If the provided version string is non-empty, onChanOpenInit should return
the version string if valid or an error if the provided version is invalid.
If the version string is empty, onChanOpenInit is expected to
return a default version string representing the version(s)
it supports.
If there is no default version string for the application,
it should return an error if provided version is empty string.

```

Reply:
ChanOpenInit -> onChanOpenInit

Reply:
onChanOpenInit -> `onChanOpenInit`